login.css

```diff
@@ -2,72 +2,46 @@ body{
 	margin:0;
 	padding:0;
 	background:white;
-	overflow: scroll;
-	overflow-x: hidden;	
 }
 
+/* Set up grid template columns and rows */
 .grid-container{
 	display: grid;
-	grid-template-columns: 15% 35% 50%;
-	grid-template-rows: auto auto auto auto ;
+	grid-template-columns: 35% 15% 50%;
+	grid-template-rows: 58px auto auto;
 	height: 100vh;
 }
 
-.img_1{
-	grid-column-start: 1;
-	grid-column-end: span 4;
-	height:300px;
-	background:red;
-}
-
-.img_1 img{
-	display:block;
-	margin-left:auto;
-	margin-right:auto;
-	height:300px;
-	width:100%;
-}
-
-.footer{
-	background-color: black;
-	grid-column-end: span 4;
-	padding-top:25px;
-	text-align: center;
-	color:black;
-	height:50px;
-	color:white;
-}
-
+/* Set up position for the nav */
 .navigation{
 	box-sizing: border-box;
-	grid-columns-start: 1;
+	grid-column-start: 1;
 	grid-column-end: span 4;
-	margin-top:-10px;
-	height:60px;
-	background-color:red;
 }
 
 
 nav{
 	width:100%;
-	background:white;
+	background-color:white;
 	overflow:auto;
-	position: static;
```

Comment:
01/08/2021: Remove the position because I don't think it needed (less confuse for me and for other)